aws-rds-integration/src/main/java/software/amazon/rds/integration/BaseHandlerStd.java

```diff
@@ -40,6 +42,18 @@ public abstract class BaseHandlerStd extends BaseHandler<CallbackContext> {
     protected static final int MAX_LENGTH_INTEGRATION = 63;
     protected static final int CALLBACK_DELAY = 6;
 
+    /** Huge database resync could potentially take a very long time. */
+    public static final Constant CREATE_UPDATE_DELAY = Constant.of()
+            .delay(Duration.ofSeconds(30))
+            .timeout(Duration.ofHours(8))
```

Comment:
I would suggest updating the schema with the timeouts. Here an example: [DBInstance Timeout](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds/blob/master/aws-rds-dbinstance/aws-rds-dbinstance.json#L592).

Reply:
Oops, I missed this message, I will follow up in a second PR.